cdap-docs/admin-manual/source/cdap-hadoop-compatibility.rst

```diff
@@ -29,6 +29,18 @@ Cloudera Distribution of Apache Hadoop (CDH)
 +================+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+
 | CDAP Version   | 5.11    | 5.10    | 5.9     | 5.8     | 5.7     | 5.6     | 5.5     | 5.4     | 5.3     | 5.2     | 5.1     |
 +----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| 5.0.0          | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **No*   | **No**  | **No**  |
```

Comment:
5.0.0 officially supports upto cdh 5.14. Also older releases support cdh 5.12+. Can you help correct those?

Reply:
sure, by older do you mean 4.3.x?